plant/plants/tests/test_views.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from django.urls import reverse
+from django.utils import timezone
+
+from plant.plants.tests.factories import PlantFactory, WateringFactory
+
+
+class TestPlantViewSet:
+    @pytest.mark.django_db()
+    def test_list_plants(self, client):
+        plants = PlantFactory.create_batch(size=5)
+        url = reverse("plants:plants-list")
+        response = client.get(url)
+        assert response.data[0]["name"] == plants[0].name
+        assert response.status_code == 200
```

Comment:
Instead of hardcoded 200 please use 
`assert response.status_code == status.HTTP_200_OK`